### DISCUSSION TITLE
Qualcomm Atheros AR9462 Test results

### DISCUSSION THREAD

**Discussion by 5jd** [vanilla-wiiu/vanilla#30] [Hardware Results]
Everything works greatly with the Qualcomm Atheros AR9462, i had no sync, input, video or audio issues whatsoever.

I've noticed the gamepad screen has a bit more latency than the real thing.

Anyways, thanks for this project that will surely revive a lot of Wii U consoles without a Gamepad! :D

**Comment by VoxelTek** [comment]
I'll add this to the compatibility table, then this issue can probably be moved to a discussion 👍🏻
